doc/design.md

```diff
@@ -231,6 +231,30 @@ Covers:
 
 Needs: impl, utest, itest
 
+## Administration Convenience Scripts
+ 
+RLS adapter provides scripts which make administration of RLS more user-friendly.
```

Comment:
It's not really the adapter. The scripts are a separate part of the project.

Reply:
Fixed